rust/perspective-viewer/src/less/viewer.less

```diff
@@ -10,6 +10,10 @@
 // ┃ of the [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0). ┃
 // ┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
 
+:host {
+    --settings-panel-z-index: 10;
```

Comment:
IMO we want to save settings for theme-able properties, not just constants. Perhaps we can add a special namespace for "private" variables.